pioneer/packages/joy-roles/src/transport.substrate.ts

```diff
@@ -727,13 +760,13 @@ export class Transport extends TransportBase implements ITransport {
           if (!roleAccount) {
             reject(new Error('failed to create role account'));
           }
-          const tx = this.api.tx.contentWorkingGroup.applyOnCuratorOpening(
-            membershipIds[0],
-            new u32(id),
-            new GenericAccountId(roleAccount as string),
-            roleStake.eq(Zero) ? null : roleStake,
-            appStake.eq(Zero) ? null : appStake,
-            applicationText
+          const tx = this.apiExtrinsicByGroup(group, 'applyOnOpening')(
+            membershipIds[0], // Member id
+            new u32(id), // Worker/Curator opening id
```

Comment:
Fixed. There were some occurences like that in `Admin.controller.tsx` too. Those weren't causing any trouble porobably because `@polkadot/api` knows how to convert any input according to the api metadata (unless the id is > 4294967295 in which case the`new u32()` would be a problem), but it's good to keep those consistent.